projects/ngx-json-schema-form/src/lib/json-schema-form.component.ts

```diff
@@ -84,10 +90,13 @@ export class JsonSchemaFormComponent implements OnChanges, OnInit {
      */
     @Input() target: '_self' | '_blank' | '_parent' | '_top' | string | null;
 
+    @Output() readonly event: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
Need to come up with better api here I think. 'event' is too generic and would be nice to have type vs 'any'

Reply:
Looked into using DOM eventing, but not simple to pass layout back on UIEvent.  Possible options are to pass back data-layout-id and then look it up in service or map dom node to layout and lookup in service.  However, getting the layout outside of the jsf component is not a simple thing.  Possibly add a EventPlugin to lookup the dom node from a service and get the layout and pass it back in the handler ass a second parameter?